packages/block-library/src/media-text/style.scss

```diff
@@ -31,15 +31,14 @@
 	vertical-align: middle;
 }
 
-.wp-block-media-text figure > div {
-	width: 100%;
+.wp-block-media-text.is-image-fill figure {
 	height: 100%;
 	min-height: 250px;
 	background-size: cover;
 }
 
-.wp-block-media-text.is-image-fill .wp-block-media-text__media {
-	height: 100%;
+.wp-block-media-text.is-image-fill figure > img {
+	display: none;
```

Comment:
The idea is correct here (ie: hide the image visually) but unfortunately this will result in the element being unavailable to assistive technologies (see [here](https://a11yproject.com/posts/how-to-hide-content/)).

I advise using an inclusive pattern to visually hidden the image but ensure it's available for screen readers. Calypso has [this component](https://github.com/Automattic/wp-calypso/tree/master/client/components/screen-reader-text) - does Gutenberg have something similar? If not look for an existing CSS class. Last resort write your own I guess.

Reply:
I've just pushed an update that makes the image visually hidden but accessible to assistive technologies.